shared/js/ui/pages/feedback.es6.js

```diff
@@ -0,0 +1,43 @@
+const Parent = window.DDG.base.Page
+const mixins = require('./mixins/index.es6')
+const parseUserAgentString = require('./../models/mixins/parse-user-agent.es6.js')
+const FeedbackFormView = require('../views/feedback-form.es6')
+const FeedbackFormModel = require('../models/feedback-form.es6')
+
+function Feedback (ops) {
+    Parent.call(this, ops)
+}
+
+Feedback.prototype = window.$.extend({},
+    Parent.prototype,
+    mixins.setBrowserClassOnBodyTag,
+    mixins.parseQueryString,
+    parseUserAgentString,
+    {
+
+        pageName: 'feedback',
+
+        ready: function () {
+            Parent.prototype.ready.call(this)
+            this.setBrowserClassOnBodyTag()
+
+            let params = this.parseQueryString(window.location.search)
+            let browserInfo = this.parseUserAgentString()
+
+            this.form = new FeedbackFormView({
+                appendTo: window.$('.js-feedback-form'),
+                model: new FeedbackFormModel({
+                    isBrokenSite: params.broken,
+                    url: decodeURIComponent(params.url || ''),
```

Comment:
@andrey-p I assume sending the entire URL is intentional? In production we only send the domain.
I don't have a strong opinion, but this approach is probably better if we want to understand the exact issue, so 👍

Reply:
Yeah, I think this will make our lives easier in the long run. There's currently also a field where the user could paste their URL, which (I think) most users also used. So this will make users' lives easier also :)